native/src/modules/common/components/RemoteContent.js

```diff
@@ -32,70 +31,65 @@ type PropType = {|
   onLoad: void => void
 |}
 
-class RemoteContent extends React.Component<PropType, StateType> {
-  state = {
-    webViewHeight: 0,
-    webViewWidth: 0
-  }
+const RemoteContent = (props: PropType) => {
+  const [webViewHeight, setWebViewHeight] = useState(0)
+  const [webViewWidth, setWebViewWidth] = useState(0)
 
-  onLayout = (event: ViewLayoutEvent) => {
+  useEffect(() => {
+    props.onLoad()
+  }, [webViewHeight])
+
+  function onLayout (event: ViewLayoutEvent) {
     const { width } = event.nativeEvent.layout
-    this.setState({ webViewWidth: width })
+    setWebViewWidth(width)
   }
 
-  onMessage = (event: WebViewMessageEvent) => {
+  function onMessage (event: WebViewMessageEvent) {
```

Comment:
Why are you using `function`s here? lambdas would be nicer

Reply:
If this is because of eslint then read this here: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#react-hooks

Reply:
Good suggestion :+1: